doc/refman/RefMan-pre.tex

```diff
@@ -1220,6 +1220,120 @@ \section*{Credits: version 8.6}
 Matthieu Sozeau and the {\Coq} development team\\
 \end{flushright}
 
+\section*{Credits: version 8.7}
+
+{\Coq} version 8.7 contains the result of refinements, stabilization of
+features and cleanups of the internals of the system along with a few
+new features. The main user visible changes are:
+\begin{itemize}
+\item New tactics: variants of tactics supporting existential variables
+  \texttt{eassert}, \texttt{eenough}, etc... by Hugo Herbelin. Tactics
+  \texttt{extensionality in H} and \texttt{inversion\_sigma} by Jason
+  Gross, \texttt{specialize with ...} accepting partial bindings by
+  Pierre Courtieu.
+\item Cumulative Polymorphic Inductive Types, allowing cumulativity of
+  universes to go through applied inductive types, by Amin Timany and
+  Matthieu Sozeau.
+\item Integration of the \texttt{Ssreflect} plugin and its documentation in the
```

Comment:
I believe the official case is SSReflect

Reply:
It would help if it were used consistently by SSReflect developers ^^

Reply:
indeed :)

Reply:
@mattam82 ?

Reply:
done